tron/serialize/runstate/dynamodb_state_store.py

```diff
@@ -247,11 +262,11 @@ def _deserialize_item(self, key: str, state: str) -> Optional[Dict[str, Any]]:
         try:
             json_key = key.split(" ")[0]
             if json_key == runstate.JOB_STATE:
-                job_data = Job.from_json(state)  # type: Dict[str, Any]
-                return job_data
+                job_data = Job.from_json(state)
+                return cast(Dict[str, Any], job_data)
             elif json_key == runstate.JOB_RUN_STATE:
-                job_run_data = JobRun.from_json(state)  # type: Dict[str, Any]
-                return job_run_data
+                job_run_data = JobRun.from_json(state)
+                return cast(Dict[str, Any], job_run_data)
```

Comment:
it'll be nice to get rid of these casts once from_json() can return the actual deserialized objects rather than a dict that we then use to reconstruct the objects :)